docs/pages/jpscore/jpscore_trains.md

```diff
@@ -5,189 +5,133 @@ tags: [jpscore, file, simulation]
 
 When this number exceeds the `agents_max` parameter, all train's doors are closed.
 
-## Transformation of coordinates
-
-While train's coordinates are **relative** to the start if the track (`track_start`), 
-Door's coordinates are **relative** to the point train's start (`train_start`) as defined in the [Train timetable](#train-timetable) file.
-
-![Coordinate systems of trains.]({{ site.baseurl }}/images/coordinates.png)
-
-Therefore, the global coordinates of the train's door `[A,B]` is calculated as follows
-
-$$
-A_{global} = track\_start + train\_start + A_{local},
-$$
-
-$$
-B_{global} = track\_start + train\_start + B_{local}.
-$$
-
-
-## Algorithm
-
-Given a `platform` with at least one `track`, the geometry will be changed every time a train arrives and leaves the platform.
-
-This will be done by projecting the train's doors on the track. With "projection" we mean along the orthogonal direction to the door.
-
-![platform]({{ site.baseurl }}/images/platform.png)
-
-Every projection point is mapped to the corresponding wall on the track.
+## Train timetable
+### Definition 
+A train is defined through the following information: 
+- `id` (int): id of the train
+- `type` (string): identifier of the train defined with the train types
+- `track_id` (int): id of the track the train arrives
+- `train_offset` (float): offset of the train to the track start
+- `reversed` (bool): false: train starts at track start, true: train starts at track end, default: false
+- `arrival_time` (float): time the train arrives
+- `departure_time` (float): time the train departs
+
+{%include note.html content="The parameter `reversed` add doors starting from track end in direction of track start!."%}
+
+### Example
+
+```xml
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<train_time_table>
+    <train id="1" type="RE"
+           track_id="2"
+           train_offset="2"
+           arrival_time="5" departure_time="15">
+    </train>
 
+    <train id="2" type="ICE"
+           track_id="3"
+           train_offset="1" reversed="true"
+           arrival_time="20" departure_time="30">
+    </train>
+</train_time_table>
 ```
- std::pair<Point, Wall >
-```
 
-Since every train's door `[A, B]`, corresponds to two intersection points `[T1, T2]` with pairs `<T1, W1>` and `<T2, W2>`, we have three cases:
+With the train defined as above, this would lead to the positioning of the train doors as seen here:
+![Schematic overview of train arrival on platform.]({{ site.baseurl }}/images/trains/traintimetable.png)
+
+## Geometry changes due to trains
+For realizing trains in the simulation, doors need to be added or removed dynamically during the simulation. 
+ 
+### Calculation of train door positions
+The following accounts when calculating the train door coordinates:
+- Distance between train start and door start is calculated along the lines
+- Distance between door start and door end is calculated as direct connection
+- Distance and width are persevered 
```

Comment:
I am not sure what you want to state here...

Reply:
changed wording a bit